doc/sphinx/language/core/modules.rst

```diff
@@ -1010,3 +1010,75 @@ subdirectories of path). See the command :cmd:`Declare ML Module` in
 
 See :ref:`command-line-options` for a more general view over the Coq command
 line options.
+
+.. _controlling-locality-of-commands:
+
+Controlling the scope of commands with locality attributes
+----------------------------------------------------------
+
+Many commands have effects that apply only within a specific scope,
+typically the section or the module in which the command was
+called. Locality :term:`attributes <attribute>` can alter the scope of
+the effect. Below, we give the semantics of each locality attribute
+but also alert that a few commands give a different meaning to the
+:attr:`local` and :attr:`global` attributes.
+
+.. attr:: local
+
+   The :attr:`local` attribute limits the effect of the command to the
+   current scope (section or module).
+
+   The ``Local`` prefix is an alternative syntax for the :attr:`local`
+   attribute (see :n:`@legacy_attr`).
+
+   .. note::
+
+      - For some commands, this is the only supported locality within
+        sections (e.g., for :cmd:`Notation`, :cmd:`Ltac` and
+        :ref:`Hint <creating_hints>` commands).
+
+      - For some other commands, this is the default locality within
+        sections even though other locality attributes are supported
+        as well (e.g., it is the case for the :cmd:`Arguments`
+        command).
```

Comment:
```suggestion
      - For other commands, this is the default locality within
        sections even though other locality attributes are supported
        as well (e.g., for the :cmd:`Arguments` command).
```

Of the two cases, "the only supported locality" and "the default locality", which is more more common?  If there's a definite preference for one or the other, maybe say "For _most_ commands"

Reply:
The two cases are not the only options (which seems implied by the "For other commands"). I've changed it into "For some" for more symmetry. There's no definite preference and the first case (only supported locality) is a limitation rather than a feature.